### ISSUE TITLE
How to download older firmware version?

### ISSUE THREAD

**Issue by MrSnoozles** [wladimir-computin/FimiX8-SDFG#18]
Hi, 

thanks for the great project. My Fimi X8 is on the latest firmware and I'm constantly getting a "System is initializing" warning and am unable to take off.
I'd now like to try if a lower firmware version for flight control (e.g. 1060C) would help. Unfortunately I don't know where to download older versions. Can anybody help?

**Comment by NIoSaT** [comment]
I think I have the same problem.
"System is preparing" and "System initializing" and unable to take off.
I also wanted to try an older firmware, but I cannot find any.

**Comment by MrSnoozles** [comment]
Yes I could. I contacted Fimi via email. They unlocked a new calibration (think it was gimbal calibration). After doing the calibration I was able to start again.  I think Fimi needed my Fimi ID to unlock the calibration. If you're successful it would be helpful if you add the missing steps here. :)
